makeflow/src/makeflow.c

```diff
@@ -2046,14 +2078,13 @@ if (enforcer && wrapper_umbrella) {
 
 	makeflow_log_started_event(d);
 
-
 	runtime = timestamp_get();
 
 	if (container_mode == CONTAINER_MODE_DOCKER) {
-            makeflow_wrapper_docker_init(wrapper, container_image, image_tar);
+    	makeflow_wrapper_docker_init(wrapper, container_image, image_tar);
 	}else if(container_mode == CONTAINER_MODE_SINGULARITY){
-            makeflow_wrapper_singularity_init(wrapper, container_image);
-        }
+    	makeflow_wrapper_singularity_init(wrapper, container_image);
```

Comment:
Unfortunately for spacing like this I have tabs equal to 4 spaces in vim and just match the current file, so particular tool. For whitespace, like tabs, I usually just remove the changes, though @batrick explained a git method for removing extraneous whitespace. I will try to find that conversations.